Source/Cryptlex.Net/Entities/Activation.cs

```diff
@@ -1,27 +1,30 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 
 namespace Cryptlex.Net.Entities
 {
     public class Activation
     {
-        public string? id { get; set; }
-        public DateTime? createdAt { get; set; }
-        public DateTime? updatedAt { get; set; }
-        public string? os { get; set; } // "windows" "linux" "macos" "android" "ios"
-        public string? osVersion { get; set; }
-        public string? hostname { get; set; }
-        public Location location { get; set; }
-        public string? vmName { get; set; }
-        public bool container { get; set; }
-        public bool offline { get; set; }
-        public string? appVersion { get; set; }
-        public string? productId { get; set; }
-        public DateTime? expiresAt { get; set; }
-        public List<ActivationMetadata>? metadata { get; set; }
-        public string? licenseId { get; set; }
-        public DateTime? lastSyncedAt { get; set; }
-        public DateTime? leaseExpiresAt { get; set; }
-        public List<ActivationMeterAttribute>? meterAttributes { get; set; }
+        public string? Id { get; set; }
+        public DateTime? CreatedAt { get; set; }
+        public DateTime? UpdatedAt { get; set; }
+        /// <summary>
+        /// "windows" "linux" "macOs" "android" "iOs"
```

Comment:
It is lowercase
https://api.cryptlex.com/v3/docs#tag/Activations/operation/get/v3/activations/%7Bid%7D

```
"windows" "linux" "macos" "android" "ios"
```